README.md

```diff
@@ -75,6 +75,8 @@ Before extending into new frontiers, we need to improve the loaders API enough t
 - [ ] Hooks for customizing the REPL, including transpilation and tab completion. Support users pasting TypeScript (or CoffeeScript or whatever) into the REPL and having just as good an experience as with plain JavaScript.
 
    - [ ] Support top-level `await` in the REPL API, if possible.
+     
+- [ ] Hook for string inputs: `--eval` CLI flag, `Worker` constructor, stdin, etc.
```

Comment:
It wouldn't be new hooks, it would just be extending our current ones to this type of input.

```suggestion
- [ ] Allow customizing string inputs: `--eval` CLI flag, `Worker` constructor, stdin, etc.
```